src/native/corehost/hostmisc/pal.unix.cpp

```diff
@@ -417,7 +417,7 @@ bool pal::get_dotnet_self_registered_dir(pal::string_t* recv)
     FILE* install_location_file = pal::file_open(install_location_file_path, "r");
     if (install_location_file == nullptr)
     {
-        trace::error(_X("The install_location file failed to open."));
+        trace::error(_X("The install_location file '%s' failed to open with error %d.", install_location_file.c_str(), errno));
```

Comment:
In another place we print out a more user friendly string version: https://github.com/dotnet/runtime/blob/017744628361a6fc45c05c75cf640292c670a8dd/src/native/corehost/json_parser.cpp#L138

Let's be consistent (I think eventually we should change this to print out both number and text).